packages/eslint-plugin-pf-codemods/src/rules/helpers/renameSinglePropOnNode.ts

```diff
@@ -0,0 +1,94 @@
+import { JSXAttribute, JSXIdentifier, JSXOpeningElement } from 'estree-jsx';
+import { RenameConfig } from './renamePropsOnNode';
+import { Rule } from 'eslint';
+
+function isRenameConfig(obj: any): obj is RenameConfig {
```

Comment:
May not be a blocker, but rather than typing this as any, wdyt about tweaking this function a little so that the param type matches what it would be passed in (`RenameConfig | string | Record<PropertyKey, never>`), changing the param name (since it won't only be an obj that can be passed), then adding an early return of false if `!hasValidObjectType`?

Also can we update the function name to something like `checkIsRenameConfig`